src/style/hero-section/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import './style.css';
+import image from './image-hero/image1.png'
+import { faSearch } from '@fortawesome/free-solid-svg-icons';
+import { Display } from '../typography/display';
+import { Body1 } from '../typography/body1';
+import { Button } from '../button/button';
+import { Label } from '../typography/LargeLabel';
+import { Separator } from '../separator/separator';
+
+export const Hero = () => {
+
+  const handleBuscarClick = () => {
```

Comment:
```suggestion
  const handleSearchClick = () => {
```

Lembrar de manter o código em inglês